cli/tests/integrations/test_auth.py

```diff
@@ -0,0 +1,52 @@
+import os
+
+from dcos import constants
+
+import pytest
+
+from .common import assert_command
+
+
+@pytest.fixture
+def env():
+    r = os.environ.copy()
+    r.update({
+        constants.PATH_ENV: os.environ[constants.PATH_ENV],
+        constants.DCOS_CONFIG_ENV: os.path.join("tests", "data", "dcos.toml"),
+    })
+
+    return r
+
+
+def test_info():
+    stdout = b'Authenticate to DCOS cluster\n'
+    assert_command(['dcos', 'auth', '--info'],
+                   stdout=stdout)
+
+
+def test_version():
+    stdout = b'dcos-auth version SNAPSHOT\n'
+    assert_command(['dcos', 'auth', '--version'],
+                   stdout=stdout)
+
+
+def test_login_no_auth():
+    stdout = b'Login successful!\n'
+    assert_command(['dcos', 'auth', 'login'], stdout=stdout)
```

Comment:
Does this set `core.dcos_acs_toke`? Can we test the "already logged in" case with:

```
dcos auth login
dcos auth login
```